src/index.html

```diff
@@ -8,6 +8,47 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div
+    class="card"
+    data-qa="card"
+  >
```

Comment:
```suggestion
  <div class="card" data-qa="card">
```
Думаю это можно написать в одну строку